mongodb.md

```diff
@@ -0,0 +1,99 @@
+# MongoDB
+
+- [Introduction](#introduction)
+- [Installation](#installation)
+    - [Starting a MongoDB server](#starting-a-mongodb-server)
+    - [MongoDB Driver](#mongodb-driver)
+    - [Install the Laravel MongoDB package](#install-the-laravel-mongodb-package)
+- [Configuration](#configuration)
+- [Features](#features)
+
+<a name="introduction"></a>
+## Introduction
+
+[MongoDB](https://www.mongodb.com/resources/products/fundamentals/why-use-mongodb) is one of the most popular NoSQL document-oriented database, used for its high write load (useful for analytics or IoT) and high availability (easy to set replica sets with automatic failover). It can also shard the database easily for horizontal scalability and has a powerful query language for doing aggregation, text search or geospatial queries. Instead of storing data in tables of rows or columns like SQL databases, each record in a MongoDB database is a document described in BSON, a binary representation of the data. Applications can then retrieve this information in a JSON format. It supports a wide variety of data types, including documents, arrays, embedded documents, and binary data.
```

Comment:
```suggestion
[MongoDB](https://www.mongodb.com/resources/products/fundamentals/why-use-mongodb) is one of the most popular NoSQL document-oriented database, used for its high write load (useful for analytics or IoT) and high availability (easy to set replica sets with automatic failover). It can also shard the database for horizontal scalability and has a powerful query language for doing aggregation, text search, or geospatial queries. Instead of storing data in tables of rows and columns like SQL databases, each record in a MongoDB database is a BSON document. BSON is a JSON-like binary representation and supports a wide variety of data types, including documents, arrays, embedded documents, and binary data.
```

Removing "easily" as it's a subjective term better suited to marketing content than documentation. I left "powerful" for the query language, though.

Removing language about applications returning BSON in a "JSON format".

Rephrased the last sentence a bit.